packages/flutter_driver/lib/src/driver/driver.dart

```diff
@@ -512,6 +513,31 @@ abstract class FlutterDriver {
     await sendCommand(SetTextEntryEmulation(enabled, timeout: timeout));
   }
 
+  /// Simulate the user posting a text input action.
+  ///
+  /// The available action types can be found in [TextInputAction]. The [sendTextInputAction]
+  /// does not check whether the [TextInputAction] performed is acceptable
+  /// based on the client arguments of the text input.
+  ///
+  /// This can be called even if the [TestTextInput] has not been [TestTextInput.register]ed.
+  ///
+  /// Example:
+  ///
+  /// ```dart
```

Comment:
We should probably update these code blocks so that they are analyzed. You can do it for just this one by adding:

```
/// {@tool snippet}
///
... The existing sample code
/// {@end-tool}
```

Reply:
yes, i updated, thanks.